wordpress-mobile/WordPress-iOS
# Jetpack Overlay - Refactor landscape layout and add RTL animation

Fixes #19172

LTR | RTL
-- | --
<img width=390 src=https://user-images.githubusercontent.com/34376330/185257134-86acf47e-68f2-44c2-bcd1-fe9582b3d02f.png> | <img width=390 src=https://user-images.githubusercontent.com/34376330/185257186-d5867668-20e7-41ce-b3a4-cfb9f77a483f.png>



This PR tweaks the landscape layout of the Jetpack overlay to better fit the content in it. 
Additionally: 
- adds a RTL animation, so that the WP logo falls on the leading edge in both orientations.
- Fixes banners and badges font, changing it from `body` to `callout`
- Tweaks the top constraint of the dismiss button to account for the gripper view

To test:
Set a LTR language
- Checkout this branch and enable the `jetpackPoweredBottomSheet` feature flag
- Tap on any "Jetpack powered" banner or badge to show the overlay
- Rotate to landscape and make sure the content is properly shown
- Make sure the animation has the wp logo at the leading edge when it finishes

Set a RTL language
- Repeat the steps
Additionally, test dark/light mode and accessibility fonts

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:
